packages/devtools_shared/lib/src/memory/leaks/constants.dart

```diff
@@ -0,0 +1,6 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Name of extension that enables leak tracking for applications.
+const leakTrackingExtensionName = 'ext.memory-leak-tracking';
```

Comment:
add this to service_extensions.dart instead. This is where all our service extensions constants are defined. This extension name will need to be defined in both DevTools and the Framework.

Also, rename to 'ext.flutter.memoryLeakTracking' to be consistent with flutter service extension naming practices. We have some code that checks whether an extension is a flutter service extension by verifying that it starts with 'ext.flutter.'